src/pages/CastBallotPage.tsx

```diff
@@ -17,11 +16,16 @@ const CastBallotPage = ({ unsetIsRecentVoterPrint }: Props) => (
         <Breadcrumbs step={4} />
         <Prose textCenter id="audiofocus">
           <h1 aria-label="Cast your ballot.">Cast your printed ballot</h1>
-          <Text narrow>
-            Before you cast your official printed ballot in the ballot box,
-            double-check your printed ballot to confirm your selections.
-          </Text>
-          <Text narrow>If you find a mistake, ask a poll worker for help.</Text>
+          <p>
+            Double-check your votes on your official printed ballot.
```

Comment:
`votes` is not the right terminology. Let's stick to `ballot`:

`Double check your official printed ballot.`